.github/workflows/api.yaml

```diff
@@ -0,0 +1,71 @@
+name: API
+
+on:
+  workflow_dispatch:
+env:
+  IMAGE_NAME: api
+  WORKING_DIRECTORY: ./
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:      
+
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+        with:
+          ref: SCV-415   # Specifies the branch to checkout
```

Comment:
This hard coded branch does not exist in the main repo.

Reply:
This is a temporary solution that allows me to test the Pre-download functionality that only exists in my own repo at the moment. We can wait to merge until this is finalized.